src/Commands/MachineToken/DeleteCommand.php

```diff
@@ -15,14 +16,41 @@ class DeleteCommand extends TerminusCommand
      * Removes a machine token from the logged-in user's account
      *
      * @name machine-token:delete
+     * @param string $machine_token_id The ID of the machine token to be deleted
+     * @throws \Terminus\Exceptions\TerminusException
      * @aliases mt:delete
      *
-     * @param machine_token_id The ID of the machine token to be deleted
      * @usage terminus machine-token:delete <machine_token_id>
      *   Removes the given machine token from the user's account
      */
     public function delete($machine_token_id) {
+        $user = $this->session()->getUser();
 
+        // Find the token. Will throw an exception if it doesn't exist.
+        $machine_token = $user->machine_tokens->get($machine_token_id);
+        if (empty($machine_token)) {
+            throw new TerminusException('There are no machine tokens with the id {id}.', compact($machine_token_id));
+        }
+        $name = $machine_token->get('device_name');
+
+        // Confirm the delete.
+        $this->confirm(
+          vsprintf('Are you sure you want to delete "%s"?', $name)
+        );
+
+        $this->log()->info(
+          'Deleting {name} ...',
+          array('name' => $name)
```

Comment:
I'd like to see the more universal syntax used (i.e. `[]` over `array()`), please.